### DISCUSSION TITLE
AntennaArray question

### DISCUSSION THREAD

**Discussion by sunskyday** [NVlabs/sionna#631] [General]
Hi,

```py
scene.tx_array = AntennaArray(antenna=Antenna("dipole", "V"),
                                                  positions=tf.Variable([[-0.15,0,0], [0.15,0,0]]))
```

There are two TX antennas, and the two antennas can only work in one pattern and polarization mode. How to define the pattern and polarization of each antenna?

**Comment by SebastianCa** [comment]
Hi @sunskyday,
that's right, the positions are the relative positions of each antenna in the array (see [API doc](https://nvlabs.github.io/sionna/api/rt.html#antenna-arrays) for details). All antennas in the array share the same properties.

Moving this thread to discussions.

**Comment by sunskyday** [comment]
I mean, antenna 1 location is[-0.15,0,0], antenna 1 pattern is set to ("dipole", "H");
antenna 2 location is [0.15,0,0], how to set antenna 2 pattern is ("iso", "V")?

**Reply by SebastianCa** [reply]
As mentioned in the API documentation, an antenna array is composed of identical antennas that are placed at different positions. Thus, it's currently not possible to have two different antenna patters in the Antenna Array.
